src/CMakeLists.txt

```diff
@@ -231,11 +230,14 @@ set(GAMEENGINE_SRC
     game/ui/msgbox.cpp
 )
 
-# Glob all the header files together to add to the project for msvc/xcode.
-if(CMAKE_CONFIGURATION_TYPES)
```

Comment:
You should probably keep the check against if its a CMAKE_CONFIGURATION_TYPES build so you only do all this when its actually needed for msvc or xcode.